src/slimeda/miss_counts.py

```diff
@@ -1,16 +1,45 @@
-def miss_counts(df):
+import pandas as pd
+def miss_counts(df, keyword=None, sparse=False, ascending=False):
     """
-    Return the number of missing values for each column in the df
+    Return the number of missing values and coresponding percentages for each column in the df. 
+    Users can also define a single number or word like "missing" as NaN in this function.
     Parameters
     ----------
     df: pd.DataFrame
         pandas dataframe
+    keyword: string or int or float
+             Default is None, a single number or string that users want to define
+             as NaN along with original NaNs
+    sparse: Boolen
```

Comment:
The usual spelling error 😄 

```suggestion
    sparse: Boolean
```